echo-plugins-test/src/main/kotlin/com/netflix/spinnaker/echo/plugins/NotificationAgentExtension.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.plugins
+
+import com.netflix.spinnaker.echo.api.events.Event
+import com.netflix.spinnaker.echo.api.events.NotificationAgent
+import org.pf4j.Extension
+
+@Extension
```

Comment:
This is just for testing right? Should it be in test rather than src? And maybe named something that denotes that it is for the test like `TestNotificationAgentExtension`?